examples/camera.py

```diff
@@ -2,7 +2,7 @@
 
 import asyncio
 
-from mavsdk import (CameraError, Mode)
+from mavsdk.camera import (CameraError, Mode)
```

Comment:
I would be tempted to just do:
```
from mavsdk import camera
```
and then access it using:
```
camera.Mode
```

That's possible, right?

Reply:
Yes that's possible, too. Or just `import mavsdk` and then use `mavsdk.camera.Mode`. I think it's really up to the user :+1:.